packages/app-accounts/src/Restorer.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+
+import accountObservable from '@polkadot/ui-keyring/observable/accounts';
+import withObservableBase from '@polkadot/ui-react-rx/with/observableBase';
+
+import { accountsQty, showIsOrAre, showPlural } from './util/accounts';
+import UploadButton from './UploadButton';
+import translate from './translate';
+
+type Props = I18nProps & {
+  accountAll?: Array<any>,
+  onChangeAccount: () => void
+};
+
+class Restorer extends React.PureComponent<Props> {
+  render () {
+    return (
+      <div className='accounts--Restorer'>
+        {this.renderData()}
+      </div>
+    );
+  }
+
+  private renderData () {
+    const { accountAll, onChangeAccount, t } = this.props;
+
+    return (
+      <div>
+        <div className='accounts--Restorer-message'>
+          {t('restorer.existing', {
+            defaultValue: `There ${showIsOrAre(accountAll)} ${accountsQty(accountAll)} saved account${showPlural(accountAll)}. Create an account or upload a JSON file of a saved account.`
```

Comment:
Instead of using string concat, which changes with values, use templates.

```
t('restorer.existing', {
  defaultValue: 'There {{quantity}} saved accounts. Create an account or upload a JSON file of a saved account.',
  replace: {
    quantity: accountsQty(accountAll)
  }
})
```

i18next support is/are & plurals as well, so no need to bring in those functions. (Which in itself is specific to the English language and therefore non-translatable)

Reply:
@jacogr I read through the following links that explain `replace`, `count`, and how to use plurals using the `_plural` postfix:
* https://www.i18next.com/translation-function/essentials
* https://www.i18next.com/translation-function/plurals

I then tried adding the following code, but regardless of what value is assigned to the  `count` property (which the docs say is the value used for plurals), it always just displays the value assigned to `defaultValue` in the UI. i.e. if `count` is 0, instead of returning the value assigned to `defaultValue_0` that is prefixed with `There are ` it instead just returns the `defaultValue` prefixed with `There is `, which isn't grammatically correct, and it does the same of just displaying the value assigned to `defaultValue` even if there are multiple accounts assigned to the `count` property, instead of returning the value assigned to `defaultValue_plural`.
```
{t('restorer.existing', {
  defaultValue: 'There is {{count}} saved account. {{instruction}}',
  defaultValue_0: 'There are {{count}} saved accounts. {{instruction}}',
  defaultValue_plural: 'There are {{count}} saved accounts. {{instruction}}',
  replace: {
    instruction: 'Create an account or upload a JSON file of a saved account.',
    count: accountsQty(accountAll)
  }
})}
```